agent/05_agent_configure.sh

```diff
@@ -277,4 +273,24 @@ fi
 
 generate_cluster_manifests
 
+if [ "$AGENT_E2E_TEST_TUI_BAD_DNS" = "true" ]; then
+  # Create a bad dns configuration by changing master-0's
+  # DNS server IP address
+  if [[ "$IP_STACK" = "v4" ]]; then
+    # from 192.168.111.1 to 192.168.111.2
+    yq -i -y '.hosts[0].networkConfig["dns-resolver"].config.server[0] = "192.168.111.2"' "${OCP_DIR}/agent-config.yaml"
+  fi
+  if [[ "$IP_STACK" = "v6" ]]; then
+    # from fd2e:6f44:5dd8:c956::1 to fd2e:6f44:5dd8:c956::2
+    yq -i -y '.hosts[0].networkConfig["dns-resolver"].config.server[0] = "fd2e:6f44:5dd8:c956::2"' "${OCP_DIR}/agent-config.yaml"
+  fi
+  if [[ "$IP_STACK" = "v4v6" ]]; then
```

Comment:
Really appreciated the intention to cover all the networking flavors, but do we plan to have CI jobs for all of them while running this test? Otherwise, it could be limited just to one networking type, if it helps for the simplification.

Reply:
I was going to only add the test for IPv4 only. So yeah, let's remove support for other networking type.